opencti-platform/opencti-graphql/src/resolvers/stixCoreObject.js

```diff
@@ -120,13 +120,13 @@ const stixCoreObjectResolvers = {
     connectors: (stixCoreObject, { onlyAlive = false }, context) => connectorsForEnrichment(context, context.user, stixCoreObject.entity_type, onlyAlive),
     importFiles: (stixCoreObject, { first, prefixMimeType }, context) => {
       const opts = { first, prefixMimeTypes: prefixMimeType ? [prefixMimeType] : null, entity_id: stixCoreObject.id };
-      return paginatedForPathsWithEnrichment(context, context.user, [`import/${stixCoreObject.entity_type}/${stixCoreObject.id}`], opts);
+      return paginatedForPathWithEnrichment(context, context.user, `import/${stixCoreObject.entity_type}/${stixCoreObject.id}`, stixCoreObject.id, opts);
     },
     pendingFiles: (stixCoreObject, { first }, context) => {
-      return paginatedForPathsWithEnrichment(context, context.user, ['import/pending'], { first, entity_id: stixCoreObject.id });
+      return paginatedForPathWithEnrichment(context, context.user, 'import/pending', stixCoreObject.id, { first });
```

Comment:
if the entity_id is not part of the options, it won't be used as a filter to list only files linked to this entity_id